libs/community/langchain_community/callbacks/tracers/wandb.py

```diff
@@ -191,14 +191,16 @@ def transform_serialized(serialized: Dict[str, Any]) -> Dict[str, Any]:
         serialized = remove_exact_and_partial_keys(serialized)
         return serialized
 
-    def transform_run(run: Dict[str, Any]) -> Dict[str, Any]:
+    def transform_run(run: Dict[str, Any]) -> Dict:
```

Comment:
```suggestion
    def transform_run(run: Dict[str, Any]) -> Dict[str, Any]:
```

Reply:
@eyurtsev because we are returning an empty dict otherwise the linter raises an error